### DISCUSSION TITLE
Does keda support scaling based on azure event grid topic

### DISCUSSION THREAD

**Discussion by varunvijayagopal** [kedacore/keda#3446] [Q&A / Need Help]
I am trying to migrate function app from azure to AKS. 
Current design is microservice will raise an event on an eventgrid topic , and we have event subrscirptions configured on event grid topic which will call a particular Azure function based on the event type.
I can't find scalers for azure event grid . can you please let me know if this is part of any other trigger or any other suggestions to implement above scenario

**Comment by tomkerkhove** [comment]
Azure Event Grid is sending events over HTTP which is not supported in KEDA core, but we have the HTTP add-on as an external scaler.

However, keep in mind that this scaler is in preview.

**Comment by varunvijayagopal** [comment]
@tomkerkhove if we use http based scaler, how can event grid send details of the event to fucntion app . any idea ?

**Reply by tomkerkhove** [reply]
It doesn't and frankly, the workload does not care where the load comes from - All it wants is to survive it
